cmd/run.go

```diff
@@ -473,6 +499,12 @@ a commandline interface for interacting with it.`,
 			}
 		}
 
+		// Teardown plugins
+		for _, jsPlugin := range plugins {
```

Comment:
Do we care if teardowns error out? Should we log after the output or are there repercussions for that? (ie. any tools that might start failing because output is slightly different)

Reply:
definitely log it

Reply:
Yes, please log any errors here